spec/lib/services/api/user_token_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe Api::UserTokenService do
+  describe ".generate_token" do
+    before do
+      @user = FactoryGirl.create(:user_with_group)
+    end
+
+    it "uses default token_ttl" do
+      user_token_service = described_class.new
+      token = user_token_service.generate_token(@user.userid, 'api')
```

Comment:
How about removing these two lines and lines 18 and 19 below and instead doing setup only once and changing what you need:

```ruby
let(:user_token_service) { described_class.new }
let(:token) { user_token_service.generate_token(@user.userid, 'api', token_ttl: token_ttl) }

context "without token_ttl set" do
   let(:token_ttl) { nil }

   it "uses the default from settings" do
     # expect....
   end
end

# ...